sdk/identity/identity/samples/interactiveBrowserCredential.ts

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { InteractiveBrowserCredential } from "../src/credentials/interactiveBrowserCredential";
+
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
+dotenv.config();
+
+/**
+ * The `InteractiveBrowserCredential` pops up a browser window where users
+ * are be able to safely authenticate with the Azure Active Directory.
```

Comment:
What's the intention behind the sample?
Is the user expected to bundle this sample using one of bundlers(rollup, webpack or parcel that we suggest)? (If so, this sample should not import dotenv, it is only meant for node I believe)

Reply:
We have an issue to separate the identity samples by folders and add the readmes, but it isn't budgeted for this release. The issue is: #10684

What would be the reasonable change to address what you say? Should we have folders for browser and node in the samples?

For this PR, Jonathan wanted a way to expose how to use this credential. I made this based on that idea! If this should be somewhere else, I can do so.

Reply:
I'd vote for a separate folder!